hudi-client/src/test/java/org/apache/hudi/table/TestMergeOnReadTable.java

```diff
@@ -186,6 +168,96 @@ public void testSimpleInsertAndUpdate() throws Exception {
     }
   }
 
+  // test incremental read does not go past compaction instant for RO views
+  // For RT views, incremental read can go past compaction
+  @Test
+  public void testIncrementalReadsWithCompaction() throws Exception {
```

Comment:
Let's make it clear that this an incremental on RO.. rename to `testIncrementalROReadsWithCOmpaction()`?

Reply:
This actually includes RT views too.

Example line 195: getRTIncrementalFiles(partitionPath); validateIncrementalFiles()